src/util/keycloak/Readme.md

```diff
@@ -0,0 +1,4 @@
+Files in this directory were copied from keycloak-react-web library.
```

Comment:
add the link to npm and to github repo for this library.

Reply:
what licence had original library ? did you follow all licence requests (like copyright notice) ?
check if we have any conflict with our licence (GPLv3)